### PR TITLE
Merge feature/working_branch into development.

### PR DESCRIPTION
**Pull request by Joseph-Burke**
The authors request to merge **feature/working_branch** into **development** following the completion of the entire Odin project assignment.

- Created models for User, Post and Comment.
- Ensured the correct relationships between models using _has_many_ and _belongs_to_
- Created migration files for each model.
- Tested relationships in the console as per instructions.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  validates :user_id, :link_url, presence: true
+
+  belongs_to :user
+  has_many :comments
+end
```
Comment (wathigo) [inline app/models/post.rb]: **REQUIRED**
* validations are present (body and title in automated tests but can be anything else) ([screenshot](validations are present (body and title in automated tests but can be anything else) (screenshot from Odin Project)) from Odin Project)

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required.
Great work guys :clap:
I have successfully created a user, post, and comments successfully.
The association between entities has been implemented in the right way.
We have one hard requirement we need to meet before marking this as done.

**REQUIRED**
Let's add body and title fields and validations to the post entity as shown in the comments :point_down: .

**Happy Hacking :computer: 
Simon Wathigo.**

File: app/models/post.rb (hunk 1 above)
Comment (Joseph-Burke) [reply app/models/post.rb]: A new migration has been exchanged with the **Post** table to remove the **link_url** column and add **title** and **body** columns. These two columns now also have presence validations, as requested.
Screenshot 2020-09-03 at 09 02 24

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Joseph-Burke and @Maryjanee 
### Good job you have done :clap: However, you need to add linter in the project and solve the errors that will be displayed [refer this article](https://github.com/microverseinc/linters-config/tree/master/ruby) - At Microverse we use github actions
### Make the changes and submit for another code review. Good luck.
- Happy hacking!

By Raymond
Slack @iamraymond

**Comment by Joseph-Burke** [comment]
.rubocop.yml and .github/workflows/linters.yml have now been added to the project, and the project is now passing all tests in GitHub Actions.

**Review by rOluochKe — APPROVED** [review]
Hello @Joseph-Burke and @Maryjanee 
### Well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**